cmd/release-controller/sync_verify_prow.go

```diff
@@ -56,9 +56,7 @@ func (c *Controller) ensureProwJobForReleaseTag(release *Release, verifyName str
 	if err != nil {
 		return nil, err
 	}
-	pj := prowutil.NewProwJob(spec, map[string]string{
-		"release.openshift.io/verify": "true",
-	}, map[string]string{
+	pj := prowutil.NewProwJob(spec, extraLabels, map[string]string{
```

Comment:
ok, I get it now.  I like the mechanism I think.

Reply:
I'm new here, but I think that I want annotations like 

`release.openshift.io/dockerImageReference` and `release.openenshift.io/image`.  I think we can add them separately, since `"release.openshift.io/analysis": "4.9.0-0.nightly-2021-07-09-201602"` will give me a good string to compare.

Reply:
We agreed on a followup